components/brave_rewards/common/policy_util.cc

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_rewards/common/policy_util.h"
+
+#include "brave/components/brave_rewards/common/pref_names.h"
+#include "build/build_config.h"
+#include "components/prefs/pref_service.h"
+
+namespace brave_rewards {
+
+bool IsDisabledByPolicy(PrefService* prefs) {
```

Comment:
It seems like we always call this with `profile->GetPrefs()`. Do you think this indicates that this is an operation on profiles, and therefore this function should just take a profile argument?

Reply:
Since Profile is defined in `chrome/browser/profiles/profile.h` passing it here would be a layer violation. Maybe it would make sense to add a wrapper that lives under `brave/browser/brave_rewards` and takes a profile argument? We don't currently have anything in /components that calls this, but it seems like it'd be good to have a way to check in this layer.

Reply:
Ah, of course, Chromium layering 😆  I'm sure this is fine as-is then.